VideoGame_Beginning

```diff
@@ -0,0 +1,17 @@
+Resaons for Argument
+
+Dataset
+
+Questions
```

Comment:
Where have Gaming sales come? Global Players to User Count. What is the role of Critic Scores? Can they get better over time through successful game sales?